plugins/modules/ipa_pwpolicy.py

```diff
@@ -179,6 +200,16 @@ def get_pwpolicy_dict(maxpwdlife=None, minpwdlife=None, historylength=None, minc
         pwpolicy['krbpwdfailurecountinterval'] = failinterval
     if lockouttime is not None:
         pwpolicy['krbpwdlockoutduration'] = lockouttime
+    if gracelimit is not None:
+        pwpolicy['passwordgracelimit'] = gracelimit
+    if maxrepeat is not None:
+        pwpolicy['ipapwdmaxrepeat'] = maxrepeat
+    if maxsequence is not None:
+        pwpolicy['ipapwdmaxsequence'] = maxsequence
+    if dictcheck is True:
+        pwpolicy['ipapwddictcheck'] = 'True'
+    if usercheck is True:
```

Comment:
The `is True` checks look wrong to me. These seem to make it impossible to turn these flags off again. Shouldn't this also be `is not None`? (And the statement in the `if` simply assigns the value?)

Also in general, it seems to me that this function can be heavily simplified by having one dictionary like:
```
options = {
    'passwordgracelimit': gracelimit,
    'ipapwdmaxrepeat': maxrepeat,
    ...
}

for option, value in options.items():
    if value is not None:
        pwpolicy[option] = value
```

Reply:
I put default value for these two options to `false` at line [183](https://github.com/ansible-collections/community.general/pull/7723/files#diff-f1b50028a82d3b0f977bb244861889823beac87d74b87d008b482ee97c57e292R183), in this case if the input value is not `True` this will become to `false`

Reply:
Fixed, and all true/false statements are replaced with for loop